aptos-node/src/network.rs

```diff
@@ -48,7 +48,15 @@ struct ApplicationNetworkHandle<T> {
 /// TODO: make this configurable (e.g., for compression)
 /// Returns the network application config for the consensus client and service
 pub fn consensus_network_configuration(node_config: &NodeConfig) -> NetworkApplicationConfig {
-    let direct_send_protocols: Vec<ProtocolId> = DIRECT_SEND.into();
+    let direct_send_protocols: Vec<ProtocolId> = if node_config.consensus.use_compression {
```

Comment:
Nice! Note, we'll need to do the same for the RPC message set below, i.e., remove `ConsensusRpcCompressed` from the list 😄